X/XDiag/build_tarballs.jl

```diff
@@ -0,0 +1,88 @@
+    SYMB_DEFS+=("-D${sym}=${sym}_64")
+done
+
+export CXXFLAGS="${SYMB_DEFS[@]}"
+
+if [[ "${target}" == *-apple-* ]]; then
+
+    # Needed to get std::visit working  
+    export MACOSX_DEPLOYMENT_TARGET=10.14
+
+    # Finding OpenMP is a bit complicated
+    cmake -D XDIAG_DISABLE_HDF5=On -DJulia_PREFIX=$Julia_PREFIX -DCMAKE_INSTALL_PREFIX=$prefix -DCMAKE_TOOLCHAIN_FILE=${CMAKE_TARGET_TOOLCHAIN} -DCMAKE_BUILD_TYPE=Release -DXDIAG_JULIA_WRAPPER=On -DJlCxx_DIR=$prefix/lib/cmake -DBLAS_LIBRARIES=${libdir}/libopenblas64_.${dlext} -DLAPACK_LIBRARIES=${libdir}/libopenblas64_.${dlext} -DOpenMP_libgomp_LIBRARY=${libdir}/libgomp.dylib -DOpenMP_ROOT=${libdir} -D OpenMP_CXX_LIB_NAMES="libgomp" -DOpenMP_CXX_FLAGS="-fopenmp=libgomp -Wno-unused-command-line-argument"  -S . -B build
+
+else
+
+    cmake -D XDIAG_DISABLE_HDF5=On -DJulia_PREFIX=$Julia_PREFIX -DCMAKE_INSTALL_PREFIX=$prefix -DCMAKE_TOOLCHAIN_FILE=${CMAKE_TARGET_TOOLCHAIN} -DCMAKE_BUILD_TYPE=Release -DXDIAG_JULIA_WRAPPER=On -DJlCxx_DIR=$prefix/lib/cmake -DBLAS_LIBRARIES=${libdir}/libopenblas64_.${dlext} -DLAPACK_LIBRARIES=${libdir}/libopenblas64_.${dlext} -S . -B build
+
+fi
+
+cmake --build build -j4
+cmake --install build
+
+"""
+
+# These are the platforms we will build for by default, unless further
+# platforms are passed in on the command line
+platforms = vcat(libjulia_platforms.(julia_versions)...)
+
+filter!(p -> (
+    (os(p) == "linux" && libc(p) != "musl" && arch(p) == "x86_64") ||
+    (os(p) == "linux" && libc(p) != "musl" && arch(p) == "aarch64") ||
+    (os(p) == "macos" && arch(p) == "x86_64") ||
+    (os(p) == "macos" && arch(p) == "aarch64") ||
+    (os(p) == "windows" && arch(p) == "x86_64")) &&
+    p.tags["julia_version"] !="1.6.3" &&
+    # p.tags["julia_version"] !="1.11.0" &&
+    p.tags["julia_version"] !="1.12.0", platforms)
```

Comment:
What's the problem with all the platforms you're excluding?

Reply:
They were numerous :). First the 32 bit platforms don't seem to work with the 64 bit Lapack interface which I strictly need. Then I focussed on the architectures I wanted to support in a first run. After quite some effort I got those to run. There were some issues in Julia 1.12 I could not resolve, but I guess att he moment that's not officially released anyway. 1.6. also had issues. Further, I have a hard time imagining someone using the package on a Free BSD platform, or with the musl libc, this is why I never tried to solve the issues there.

Reply:
If you want to build only for 64-bit platforms you can filter `nbits(p) == 64`, which is much simpler than listing all archs one by one